02-Logging/01-Displaying-Status-Information.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>We also allow for logging properties. The syntax is <code>(obj, property_name, flag)</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Yes, but the tutorials shouldn't be an exhaustive list of all features. We should introduce a small and carefully chosen set of concepts at one time to avoid cognitive overload. Users can find this in the reference documentation - or if it is a really important concept we can introduce it on its own in a later tutorial section.</p>

Reply:
Good point.